hardware/detect.py

```diff
@@ -927,8 +927,7 @@ def main():
     args = parse_args(sys.argv[1:])
 
     hrdw = []
-    areca_hrdw = areca.detect()
-    hrdw.append(areca_hrdw)
+    hrdw.extend(areca.detect())
```

Comment:
Do you want to check the result for None? otherwise you may end up calling extend(None)

Reply:
Yeah but append(none) was adding an empty () item while extend(None) do nothing.
This makes a oneliner easier to integrate and duplicate for other items if we want.

Reply:
```
>>> x = []
>>> x.extend(None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'NoneType' object is not iterable
```

Am I misunderstanding something?

Reply:
Looks like I missed something then

Reply:
Yeah, I misunderstood extend(). I simplified with a simple filter() call

Reply:
is it fine for you @dtantsur ?